OpenTabletDriver.Daemon/DriverDaemon.cs

```diff
@@ -84,6 +85,19 @@ public DriverDaemon(Driver driver)
 
         private bool debugging;
 
+        /// <summary>
+        /// Checks for any problematic processes running on the user's computer that may
+        /// impair function or detection of tablets.
+        /// </summary>
+        private void CheckForProblematicProcesses()
+        {
+            if (SystemInterop.CurrentPlatform == PluginPlatform.Windows)
+            {
+                if (Process.GetProcessesByName("vgc").Length > 0)
```

Comment:
Always use `.Any()` as it checks the enumerator itself providing better performance in (most) cases
```suggestion
                if (Process.GetProcessesByName("vgc").Any())
```